tools/astro-vscode/package.json

```diff
@@ -24,7 +24,7 @@
     "onLanguage:astro"
   ],
   "dependencies": {
-    "astro-languageserver": "file:../astro-languageserver"
```

Comment:
Fixes a changesets warning about this being a problem (if we need this I can revert it, but this probably shouldn’t get published, right?)